src/main/java/org/wildfly/security/auth/provider/ldap/OtpCredentialLoader.java

```diff
@@ -73,10 +80,14 @@ public ForIdentityLoader(DirContextFactory contextFactory, String distinguishedN
         }
 
         @Override
-        public SupportLevel getCredentialAcquireSupport(String credentialName) {
-            if ( ! OtpCredentialLoader.this.myCredentialName.equals(credentialName)) {
+        public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential> credentialType, final String algorithmName) {
+            if (credentialType != PasswordCredential.class) {
                 return SupportLevel.UNSUPPORTED;
             }
+            if (algorithmName == null) {
```

Comment:
I'm just going to amend the commit; there's probably no value in retaining this mistake in history :)